### PR TITLE
lint script fix for windows

### PR DESCRIPTION
**Pull request by rkmr039**
"lint": "npm-run-all --aggregate-output --continue-on-error --parallel 'lint:!(fix)'", --> "lint": "npm-run-all --aggregate-output --continue-on-error --parallel lint:!(fix)",

### CODE REVIEW

**Review by swarajpure — COMMENTED** [review]
Thanks for taking this issue!

Are you sure this command is working fine on linux/macOS?

**Comment by rkmr039** [comment]
Testing on Ubuntu

**Comment by rkmr039** [comment]
> Thanks for taking this issue!
> 
> Are you sure this command is working fine on linux/macOS?

Tested in Ubuntu, working fine.
Not sure about MacOs

File: package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
   "scripts": {
     "dev": "ember serve --port=80",
     "build": "ember build --environment production && cp _redirects dist/_redirects",
-    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel 'lint:!(fix)'",
+    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel \"lint:!(fix)\"",
```
Comment (swarajpure) [inline package.json:16]: Thanks for the change!

File: package.json [hunk 2]
```diff
@@ -66,5 +66,8 @@
   },
   "ember": {
     "edition": "octane"
+  },
+  "dependencies": {
+    "glob": "^7.1.6"
```
Comment (swarajpure) [inline package.json:71]: Where is this coming from? 🤔

Comment (rkmr039) [reply package.json:71]: \"lint:!(fix)\" part will be resolved by glob only

Comment (rkmr039) [reply package.json:71]: [Glob-package](https://www.npmjs.com/package/glob)

**Review by swarajpure — APPROVED** [review]
Was a very much needed fix! Thanks 🙂 

Congratulations on your first pull request in Real Dev Squad! 🥳

**Pull request merged**